src/components/atoms/UserAvatar/UserAvatar.scss

```diff
@@ -9,6 +9,10 @@
     height: 100%;
     width: 100%;
     border-radius: 50%;
+    object-fit: cover;
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
```

Comment:
It's probably also worth adding a background color here as a fallback in case there are problems loading the image.